packages/react-component-library/src/components/ClassificationBar/ClassificationBar.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { StoryFn, Meta } from '@storybook/react'
+import { ClassificationBar } from './ClassificationBar'
```

Comment:
We add a line break between external and internal exports.
```suggestion

import { ClassificationBar } from './ClassificationBar'
```